### DISCUSSION TITLE
Thank you for the update

### DISCUSSION THREAD

**Discussion by s0metimes** [novemberi/clover#12] [General]
Hello! 
I am getting helped with clover for searching the unity assets which are connected to the script I am looking into.
I am also surprised on the update recently done.
Seeing results by 'peak' is amazing and also, it is really good to look around assets (scenes, prefabs etc.) in vscode as it is quite annoying to go back to unity and wait for its compiling.

Thank you.
As soon as I finish on the current project, I would like to attribute for clover.
I am thinking that it would be better if clover supports for searching and linking to the asset by meta reference, as the function available in Visual Studio.

**Comment by dunward** [comment]
Sorry to late check your opinion.
I didn't try to use Visual Studio, is the function searching similar Rider method Usage?
